### DISCUSSION TITLE
Treesitter invalid node error for `help` filetype

### DISCUSSION THREAD

**Discussion by MicEsp22** [nvim-treesitter/nvim-treesitter#5883] [Q&A]
Hi everyone,

Lately, I’m getting invalid node error from treesitter when opening an help document with :h.

See this picture:
![Prompt dei comandi - nvim 08_01_2024 15_35_12](https://github.com/nvim-treesitter/nvim-treesitter/assets/60217734/5a38fa5c-a99f-4e1e-82e9-4d9214dd559c)

Anyone has (had) the same issue?
Might it be an update of treesitter that broke something?

**Comment by clason** [comment]
`:TSInstall vimdoc`; please read the documentation and check previous (also closed) issues.

**Reply by MicEsp22** [reply]
Thanks @clason . I'm sorry, I did serach for previously opened discussion obout this topic but the seach didn't give me anything relevant.. Thanks for replying.

**Reply by clason** [reply]
Also search issues, not just discussions.

**Reply by MicEsp22** [reply]
I'll do next time.
